thriftbp/server.go

```diff
@@ -46,6 +46,11 @@ type ServerConfig struct {
 	// If not set none of the requests will be sampled.
 	ReportPayloadSizeMetricsSampleRate float64
 
+	// Optional, used only by NewBaseplateServer.
+	//
+	// Report the number of clients connected to the server.
+	ReportServerConnectionCount bool
```

Comment:
1. since this is part of `thriftbp.ServerConfig`, the `Server` part in the name is kinda redundant, probably should rename this to `ReportConnectionCount`.
2. Please also mention the metrics path in the comments (e.g. "Report the number of clients connected to the server as RuntimeGauge of thrift.connections")